index.js

```diff
@@ -0,0 +1,3 @@
+aaa = bbb;
+bbb = ccc;
+ccc = ddd;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ddd' is not defined  no-undef